ImprovedAnvil/lib/items/01_copy_new.tph

```diff
@@ -785,6 +785,58 @@ COPY ~ImprovedAnvil/resources/items/new/S!potn21.itm~ ~override/S!potn21.itm~
 SAY NAME2     @10304
 SAY DESC      @10305
 
+COPY ~ImprovedAnvil/resources/items/new/S!blun04.itm~ ~override/S!blun04.itm~
```

Comment:
Haven't you deleted some of these items from the repository in the prior commits from this pull request? Commits with the commentary "Delete ..."

Reply:
Yes I took them out because of the request to do 1 item at a time, which I now interpret "item" to mean "change".  I'll wait to upload them until you're ready.

Reply:
Just to clarify what I meant. For example you want to add one/several items to the randomiser. That a _single_ logical item. That means it's OK to include the items themselves, their installation, changes to respective dialogues and the new text lines into a single commit. This is just a sensible approach, one atomic commit for one piece of change. Sorry if I didn't explain this better before.